### DISCUSSION TITLE
How many players are there in a standard men's field lacrosse team?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#233] [Galaxy Trivia] [answered]
**Hint:** 🕙

**Comment by LauraBigoni** [comment, marked as answer]
10 players

**Reply by ParanoidUser** [reply]
You nailed it! Congratulations on your well deserved [Galaxy Brain](https://github.com/LauraBigoni?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 4 answers), and don't hesitate to post your own questions to the community. 😊🎉
